### DISCUSSION TITLE
access to remix brand logo

### DISCUSSION THREAD

**Discussion by MadadiSaeed** [remix-run/remix#5330] [Q&A] [answered]
Hi. I want to create high resolution wallpaper for my desktop from remix logo. So If these resources are free for personal use, please make them available to download. specially if you have template file for this [https://dev-to-uploads.s3.amazonaws.com/i/i90palgvmevg7ocgaae1.png](https://dev-to-uploads.s3.amazonaws.com/i/i90palgvmevg7ocgaae1.png). Thanks.

**Comment by machour** [comment, marked as answer]
Hey

Right-click the logo on the https://remix.run website

**Reply by MadadiSaeed** [reply]
Thanks for the response.
